cmd/env.go

```diff
@@ -293,6 +332,33 @@ the name of an environment will also update the directory structure in
   ks env set us-west/staging --namespace=staging
 
   # Updates both the name and the URI of the environment 'us-west/staging'.
-  # Updating the name will update the directory structure in 'environments'
-  ks env set us-west/staging --uri=http://example.com --name=us-east/staging`,
+  # Updating the name will update the directory structure in 'environments'.
+  ks env set us-west/staging --uri=http://example.com --name=us-east/staging
+  
+  # Updates URI of the environment 'us-west/staging' based on the server URI
+  # in the context 'staging-west' in your kubeconfig file.
+  ks env set us-west/staging --context=staging-west`,
+}
+
+func commonEnvFlags(flags *pflag.FlagSet) (uri, namespace, context string, err error) {
```

Comment:
Two things.

1. I know I suggested earlier that we should make `--context` and `--uri` mutually exclusive, but now I'm not so sure. Now I'd like to propose that we do whatever kubectl does. What do you think? Sorry for being slightly schizoid about this. I keep forgetting that we have a reference implementation for these flags.
1. If we keep this mutual exclusion, I think it's worth mentioning a comment of this function that `--uri` and `--context` are mutually exclusive, but `--context` and `--namespace` are not, as well as _why_.

Reply:
1. So, client-go doesn't have a flag `uri`, but it has a `server` flag. It doesn't complain if we try to provide both flags but the server flag seems to be ignored.
2. Using your message from below.

Reply:
Ok, so we should probably have a `--server` flag, then, is what I'd say.

Reply:
Talked offline, we're going to punt this to a follow-up PR to change `uri` references to `server` across the board, including the environment's `spec.json` file.